packages/features/calendar-cache/calendar-cache.repository.ts

```diff
@@ -19,8 +18,8 @@ function parseKeyForCache(args: FreeBusyArgs): string {
   // Ensure that calendarIds are unique
   const uniqueItems = uniqueBy(args.items, ["id"]);
   const key = JSON.stringify({
-    timeMin: getTimeMin(args.timeMin),
-    timeMax: getTimeMax(args.timeMax),
```

Comment:
Repo won't modifiy dates anymore. I see it as an anti-pattern

Reply:
Me too man, me too.